pkg/render/markdown.go

```diff
@@ -42,7 +42,9 @@ func matchFragmentLink(s string) string {
 }
 
 func (r Markdown) File(ctx context.Context, fr *bincapz.FileReport) error {
-	markdownTable(ctx, fr, r.w, tableConfig{Title: fmt.Sprintf("## %s [%s]", fr.Path, mdRisk(fr.RiskScore, fr.RiskLevel))})
+	if len(fr.Behaviors) != 0 {
```

Comment:
NOTE: This further breaks `--omit-empty`, but we should get rid of it anyways, IMHO (see #348)